commit ca662958218f4484a89be94015066ff6a14875a8
Author: Alexey Tikhonov <atikhono@redhat.com>
Date:   Wed Apr 15 09:42:34 2026 +0200

    Add missing include
    
    Original patch f3af8c89af656767333410b0e94da9288dd8ade8 didn't include
    "config.h" that provides `HAVE_PTHREAD_EXT`
    It works in some branches accidentally because of transitive include
    via "sss_cli.h" but that's fragile (and in some branches "sss_cli.h"
    doesn't include "config.h")
    
    Reviewed-by: Tomáš Halman <thalman@redhat.com>
    (cherry picked from commit a809b9236250e6f20e9a9ff1452708cd288b705f)

diff --git a/src/sss_client/autofs/sss_autofs.c b/src/sss_client/autofs/sss_autofs.c
index f5986767f..45e2ce460 100644
--- a/src/sss_client/autofs/sss_autofs.c
+++ b/src/sss_client/autofs/sss_autofs.c
@@ -18,6 +18,8 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+#include "config.h"
+
 #include <errno.h>
 #include <stdlib.h>
 #include <stdatomic.h>
